docs/by-example/client.rst

```diff
@@ -35,12 +35,8 @@ respond, the request is automatically routed to the next server:
     >>> connection = client.connect([invalid_host, crate_host])
     >>> connection.close()
 
-If no ``servers`` are given, the default one ``http://127.0.0.1:4200`` is used:
-
-    >>> connection = client.connect()
-    >>> connection.client._active_servers
-    ['http://127.0.0.1:4200']
-    >>> connection.close()
+If no ``servers`` are supplied to the ``connect`` method, the default address
+``http://127.0.0.1:4200`` is used.
```

Comment:
There is no server at `http://127.0.0.1:4200`. This test case just didn't fail because `connect()` did not raise an exception up until now.

Now, there is no longer a way to validate connecting to the default address per doctest file, because there is no server listening at `http://127.0.0.1:4200`. Because the core information is still viable, all what's left is pure prose, rephrased a bit.